src/config.rs

```diff
@@ -79,7 +83,15 @@ pub struct Config {
     pub(crate) target: LevelFilter,
     pub(crate) target_padding: TargetPadding,
     pub(crate) location: LevelFilter,
+    #[cfg(feature = "chrono")]
+    pub(crate) time_format: Cow<'static, str>,
+    #[cfg(feature = "chrono")]
+    pub(crate) time_offset: FixedOffset,
+    #[cfg(feature = "chrono")]
+    pub(crate) time_local: bool,
```

Comment:
Then only the `time_local` field would be specific to chrono (and I don't really see a way to remove it).